test/jasmine/tests/annotations_test.js

```diff
@@ -775,7 +775,7 @@ describe('annotations autorange', function() {
             });
         })
         .then(function() {
-            _assert('auto rng / big tx', [-0.22, 3.57], [0.84, 3.365]);
+            _assert('auto rng / big tx', [-0.22, 3.59], [0.84, 3.365]);
```

Comment:
Doesn't matter for CI, and has nothing to do with this PR, but this change makes this test pass on my Mac.